lib/zendesk_apps_support/package.rb

```diff
@@ -42,22 +51,36 @@ def validate
           errors << Validations::Banner.call(self)
         end
 
-        errors.flatten!
+        errors.flatten!.compact!
       end
     end
 
-    def app_js
-      read_file('app.js')
+    def validate!(marketplace: true)
+      errors = validate(marketplace: marketplace)
+      raise errors.first if errors.any?
```

Comment:
were we already only raising the first error? Would probably be good to somehow show all the information at once if possible.